test/common.go

```diff
@@ -5,15 +5,16 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
-const maxValidNameLength = 62
+const maxValidNameLength = 63
 
-var TestOversizedNameObj = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+var TestOversizedNameObj = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijkl" // len=64 chars
+var TestMaxsizedNameObj = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijk"   // len=63 chars
 
 // CreateRandomValidTestName functions creates a valid lenght name
 func CreateRandomValidTestName(name string) string {
 	randomName := name + uuid.NewV4().String()
-	if len(randomName) > 62 {
-		return randomName[:61]
+	if len(randomName) > maxValidNameLength {
```

Comment:
Use of maxValidNameLength variable rather hard coding.

Reply:
I don't have have much experience with this, but this file seems to be generated automatically! So I think this should be generated automatically if not done so.

Reply:
This file seems to be generated by counterfeiter (may be this one [link](https://github.com/maxbrunsfeld/counterfeiter)) to generate test files.  To me the auto-generated code is not very well.  See first version of this file - https://github.com/fabric8-services/fabric8-wit/pull/1175/files#diff-fd4f597ee820067bb15738e07473928b it has added "maxValidNameLength" but never used though it can be used in the same file.

I have make changes in this file to use maxValidNameLength appropriately.  Also changed value of TestOversizedNameObj to have in-valid name which has exact one extra char (i.e. 64-chars).

So though it's generated before, I don't like auto-generate version and changed which suite best for code under test :-)

Reply:
the only thing is that if someone else re-generates this file all the manual changes will be lost, so for this particular change I don't have strong opinion :+1:

Reply:
We moved away from counterfeiter a long time ago. Unless someone added it again? I think the comment is just left overs.